keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -146,10 +138,24 @@ _This section must be completed when targeting alpha to a release._
     - Feature gate name: PreferNominatedNode
     - Components depending on the feature gate: kube-scheduler
 
+* **Does enabling the feature change any default behavior?**
+  Yes. The pod with the nominated node set will be evaluated first in any scheduling cycle,
+  this is only the default process logic that is handled by scheduler, end-user will not
+  and need not aware of any difference.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes. This could be opt-out by the feature gate, once switch off, it will fall back
```

Comment:
Sorry - I don't understand the first sentence...

Reply:
reword a little bit, hope it will help.